pkg/v1/providers/bootstrap-kubeadm/v1.1.3/bootstrap-components.yaml

```diff
@@ -6278,7 +6278,6 @@ spec:
         - --leader-elect
         - --metrics-bind-addr=localhost:8080
         - --feature-gates=MachinePool=${EXP_MACHINE_POOL:=false},KubeadmBootstrapFormatIgnition=${EXP_KUBEADM_BOOTSTRAP_FORMAT_IGNITION:=false}
-        - --bootstrap-token-ttl=${CAPBK_BOOTSTRAP_TOKEN_TTL:=15m}
```

Comment:
is this no longer relevant/applicable?

Reply:
Yes it is removed from the template and is no longer applicable.